docs/source/conf.py

```diff
@@ -171,7 +171,11 @@
 # "default.css" will overwrite the built-in "default.css".
 html_css_files = ["css/termynal.css", "css/termynal_custom.css", "css/custom.css"]
 
-html_js_files = ["js/termynal.js", "js/custom.js"]
+html_js_files = [
+    "js/termynal.js",
+    "js/custom.js",
+    "js/require.js",
```

Comment:
We need `require.js` for plotly to work with sphinx. See [here](https://github.com/readthedocs/sphinx_rtd_theme/issues/788).